### PR TITLE
ലിങ്ക് ഉണ്ടാക്കാൻ വേണ്ടി ഒരു താൾ

### PR DESCRIPTION
**Pull request by Jobin-S**

### CODE REVIEW

**Comment by Jobin-S** [comment]
@subins2000 I didn't work on vue js yet. When I saw this issue I got interested to work on it. 
image
On my local, it is showing the /create route.
but on the preview link it is not

File: src/views/CreateView.vue [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        .join("-");
+      this.newLink = slug + ".കിട്ടും.com";
+    },
+    async copyURL(mytext) {
+      try {
+        await navigator.clipboard.writeText(mytext);
+        alert("Copied");
+      } catch ($e) {
+        alert("Cannot copy");
+      }
```
Comment (subins2000) [inline src/views/CreateView.vue:34]: @Jobin-S Better to show some text message rather than an alert. Would you be able to make another PR?

Comment (Jobin-S) [reply src/views/CreateView.vue:34]: text message as toast?

Comment (subins2000) [reply src/views/CreateView.vue:34]: doesn't have to be a full advanced toast, just anything other than an alert box will do
